src/node/rpc/member_frontend.h

```diff
@@ -847,58 +847,57 @@ namespace ccf
         .set_require_client_signature(true)
         .install();
 
-      auto get_vote =
-        [this](ReadOnlyEndpointContext& args, nlohmann::json&& params) {
-          if (!check_member_active(args.tx, args.caller_id))
-          {
-            return make_error(HTTP_STATUS_FORBIDDEN, "Member is not active");
-          }
+      auto get_vote = [this](ReadOnlyEndpointContext& args, nlohmann::json&&) {
```

Comment:
Nit: Not a task for this PR, but if we ever make a change to our clang-format I'd really like to fix this behaviour. Minor reflowing shouldn't the indentation of so many lines.

Reply:
Things may or may not get better once we go clang*-10.